client/src/components/profile/components/internet.tsx

```diff
@@ -107,46 +90,38 @@ class InternetSettings extends Component<InternetProps, InternetState> {
       state: 'error',
       message: t('validation.invalid-url')
     };
-  }
+  };
 
-  createHandleChange =
+  const createHandleChange =
     (key: keyof Socials) => (e: React.FormEvent<HTMLInputElement>) => {
       const value = (e.target as HTMLInputElement).value.slice(0);
-      return this.setState(state => ({
-        formValues: {
-          ...state.formValues,
-          [key]: value
-        }
-      }));
+      setFormValues({
+        ...formValues,
+        [key]: value
+      });
     };
 
-  isFormPristine = () => {
-    const { formValues, originalValues } = this.state;
+  const isFormPristine = () => {
```

Comment:
```suggestion
  const isFormPristine = () => {
    const originalValues = { githubProfile, linkedin, twitter, website };
```
This is all we need.